packages/storybook/stories/components/tabs/tabs.stories.ts

```diff
@@ -0,0 +1,197 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-tabs';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/Navigation/Tabs',
+  component: 'ods-tabs',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (args) => html`
+<ods-tabs>
+  ${unsafeHTML(args.content)}
+</ods-tabs>
+  `,
+  argTypes: orderControls({
```

Comment:
- I think it will be better to set a static content and add `isDisabled` and `isSelected` argsTypes for each tab. 
Or add an others story demo

Reply:
We discussed about this yesterday with François too. We agreed that only activating one specific tab is acceptable as long as it is explained in the control description. WDYT?